config/config.go

```diff
@@ -119,6 +122,8 @@ func DefaultConfigMainnet() *Config {
 	conf.GRPC.BasicAuthCredential = ""
 	conf.GRPC.Gateway.Enable = false
 	conf.GRPC.Gateway.Listen = "127.0.0.1:8080"
+	conf.JSONRPC.Enable = false
```

Comment:
Please update the `example_config.toml`

Reply:
[3e1a4cad865e79547daba398fb9ad200cfbf07a2](https://github.com/pactus-project/pactus/pull/1183/commits/3e1a4cad865e79547daba398fb9ad200cfbf07a2)